content/zh/docs/home/_index.md

```diff
@@ -1,42 +1,94 @@
 ---
 title: Kubernetes 文档
-layout: docsportal_home
 noedit: true
-cid: userJourneys
-css: /css/style_user_journeys.css
-js: /js/user-journeys/home.js, https://use.fontawesome.com/4bcc658a89.js
-display_browse_numbers: true
+cid: docsHome
+layout: docsportal_home
+class: gridPage
 linkTitle: "主页"
 main_menu: true
 weight: 10
+hide_feedback: true
 menu:
   main:
     title: "文档"
     weight: 20
     post: >
       <p>通过演练，示例和参考文档了解如何使用 Kubernetes。你甚至可以<a href="/editdocs/" data-auto-burger-exclude>帮助贡献文档</a>！</p>
+overview: >
+  <!-- Kubernetes is an open source container orchestration engine for automating deployment, scaling, and management of containerized applications. The open source project is hosted by the Cloud Native Computing Foundation (<a href="https://www.cncf.io/about">CNCF</a>). -->
+  Kubernetes 是一个开源容器编排引擎，用于容器化应用的自动化部署、扩展和管理。该项目托管在 <a href="https://www.cncf.io/about">CNCF</a>。
+<!-- cards:
+- name: concepts
```

Comment:
Do you have any reference for such changes? Like the layout of the file?

Reply:
Seems work fine by netlify preview.
It's strange that the doc in release-1.14 the md looks fine, while in master the content is missing.
```
https://raw.githubusercontent.com/kubernetes/website/release-1.14/content/zh/docs/home/_index.md
vs
https://raw.githubusercontent.com/kubernetes/website/master/content/zh/docs/home/_index.md
```
Maybe some pr changed this by mistake?

Reply:
Also the file in master looks like in old style format.